dashboard/config/routes.rb

```diff
@@ -751,7 +755,9 @@
   get 'foorm/preview/:name', to: 'foorm_preview#name', constraints: {name: /.*/}
   get 'foorm/preview', to: 'foorm_preview#index'
 
-  get 'foorm/editor', to: 'foorm_editor#index', constraints: {name: /.*/}
+  get 'foorm/editor', to: 'foorm_form_editor#index'
```

Comment:
Is this route just here for backwards compatibility?  If so, might be good to add a comment?

Reply:
Yep, I can add a comment. I think the more common case will be wanting to edit a form vs. editing a library, so wanted to keep it as the default.